plugins/modules/one_vnet.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024, Alexander Bakanovskii <skottttt228@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: one_vnet
+short_description: Manages OpenNebula virtual networks
+version_added: 9.4.0
+author: "Alexander Bakanovskii (@abakanovskii)"
+requirements:
+  - pyone
+description:
+  - Manages OpenNebula networks
+attributes:
+  check_mode:
+    support: partial
+    details:
+      - Note that check mode always returns C(changed=true) for existing networks, even if the network would not actually change.
+  diff_mode:
+    support: none
+options:
+  id:
+    description:
+      - A O(id) of the network you would like to manage.
+      - If not set then a new network will be created with the given O(name).
+    type: int
+  name:
+    description:
+      - A O(name) of the network you would like to manage.  If a network with
+      - the given name does not exist it will be created, otherwise it will be
+      - managed by this module.
+    type: str
+  template:
+    description:
+      - A string containing the network template contents.
+    type: str
+  state:
+    description:
+      - V(present) - state that is used to manage the network.
+      - V(absent) - delete the network.
+    choices: ["present", "absent"]
+    default: present
+    type: str
+
+extends_documentation_fragment:
+  - community.general.opennebula
+  - community.general.attributes
+'''
+
+EXAMPLES = '''
+- name: Fetch the NETWORK by id
+  community.general.one_vnet:
+    id: 0
+  register: result
+
+- name: Fetch the TEMPLATE by name
+  community.general.one_vnet:
+    name: opennebula-bridge
+  register: result
+
+- name: Create a new or update an existing NETWORK
+  community.general.one_vnet:
+    name: bridge-network
+    template: |
+      VN_MAD  = "bridge"
+      BRIDGE  = "br0"
+      BRIDGE_TYPE  = "linux"
+      AR=[
+        TYPE  = "IP4",
+        IP    = 192.0.2.50,
+        SIZE  = "20"
+      ]
+      DNS     = 192.0.2.1
+      GATEWAY = 192.0.2.1
+
+- name: Delete the NETWORK by id
+  community.general.one_vnet:
+    id: 0
+    state: absent
+'''
+
+RETURN = '''
+id:
+    description: network id
```

Comment:
```suggestion
    description: The network ID.
```
Similar below, `description` should always have complete sentences in them.